content/en/monitoring/notification-manager.md

```diff
@@ -355,9 +357,9 @@ spec:
 EOF
 ```
 
->Slack token is the OAuth Access Token or Bot User OAuth Access Token when you create a slack app. This app must have the scope chat:write. The user who created the app or bot user must be in the channel which you want to send notification to.
+> Slack token is the OAuth Access Token or Bot User OAuth Access Token when you create a slack app. This app must have the scope chat:write. The user who created the app or bot user must be in the channel which you want to send notification to.
```

Comment:
The user who created the app or bot user must be in the channel which you want to send notification to. =>
The user who creates the app or bot user must be in the channel to which you want to send notification.